src/screens/OrganizationActionItems/OrganizationActionItems.tsx

```
@@ -138,46 +148,84 @@ function organizationActionItems(): JSX.Element {
     () => debounce((value: string) => setSearchTerm(value), 300),
     [],
   );
 
-  // Trigger refetch on sortBy or status change
+  /**
+   * Apply client-side filtering and sorting like the category component
+   */
   useEffect(() => {
-    actionItemsRefetch();
-  }, [sortBy, status, actionItemsRefetch]);
+    if (actionItemsData && actionItemsData.actionItemsByOrganization) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)